package.json

```diff
@@ -2,6 +2,7 @@
   "private": "true",
   "scripts": {
     "format": "prettier . --write",
+    "lint": "eslint .",
```

Comment:
Can you please put the `lint` command before the `format` comment? It makes the formatting look a bit better.

Reply:
Done, haha, not sure how I put it there.